.github/workflows/dotnet-desktop.yml

```
@@ -1,115 +1,397 @@
-    - name: Upload build artifacts
-      uses: actions/upload-artifact@v4
-      with:
-        name: MSIX Package
-        path: ${{ env.Wap_Project_Directory }}\AppPackages
+  # Create GitHub Release (only on main branch and Release builds)
+  release:
+    name: Create Release
+    needs: build
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    runs-on: ${{ github.event.inputs.runner_os || 'windows-latest' }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)